mifosng-android/src/main/res/values/strings.xml

```diff
@@ -799,6 +799,7 @@
     <string name="document_dialog_fragment_loading_message">Uploading document</string>
     <string name="data_table_row_dialog_loading_message">Adding entry to database</string>
     <string name="savings_account_approval_fragment_loading_message">Approving savings account</string>
+    <string name="please_enter_the_credentials">Please Enter the Credentials</string>
```

Comment:
```suggestion
    <string name="error_enter_credentials">Please Enter the Credentials</string>
```

Reply:
Yes, you are right. I will update it accordingly.